stripe/_account_capability_service.py

```diff
@@ -1,43 +1,25 @@
 # -*- coding: utf-8 -*-
 # File generated from our OpenAPI spec
+from stripe._account_capability_list_params import AccountCapabilityListParams
```

Comment:
these should all probably go in an `if TYPE_CHECKING` block. Won't need changes to the main body of the service, since the types are already in a string